packages/app/src/runs/modals/CloudConnectModals.spec.tsx

```diff
@@ -29,10 +39,43 @@ describe('<CloudConnectModals />', () => {
     })
   }
 
-  it('shows the select org modal when orgs are added', () => {
-    mountDialog()
-    cy.contains(defaultMessages.runs.connect.modal.selectProject.connectProject).should('be.visible')
+  context('has no organization', () => {
+    beforeEach(() => {
+      mountDialog(false, false)
+    })
+
+    it('shows the create/select org modal when orgs are added', () => {
+      cy.contains(defaultMessages.runs.connect.modal.createOrg.button).should('be.visible')
+
+      cy.percySnapshot()
+    })
+  })
+
+  context('has organizations', () => {
+    context('with no projects', () => {
+      beforeEach(() => {
+        mountDialog(true, false)
```

Comment:
`mountDialog(true, false)` seems kind of hard to understand - I wonder if this should be an object of options?